.clang-format

```diff
@@ -15,14 +15,19 @@ AlignConsecutiveAssignments: false
 AlignConsecutiveDeclarations: false
 AlignEscapedNewlines: DontAlign
 AlignOperands:   true
+
 AlignTrailingComments: false
+# > clang-format-13
+#    Kind: Leave
+#    OverEmptyLines: 2
```

Comment:
Can we uncomment this? Below we also have the lambda configs which assume clang-format-13+

Reply:
This looks horrible when turned on, and touches nearly all comments. I'll leave it as is